DCOS/fluentd-testing/run_fluentd_tests.ps1

```diff
@@ -0,0 +1,13 @@
+$TESTS_DIR = Split-Path -Parent $MyInvocation.MyCommand.Path
+pushd "$TESTS_DIR"
+$results = Invoke-Pester -PassThru
+$FailedTestTally = $results.FailedCount
+if ($FailedTestTally -eq 0) {
+	write-output "All tests passed successfully"
+	exit 0
+}
+else {
+	write-output "Some tests failed. Number of failed tests is: $FailedTestTally"
+	exit 1
+}
+popd
```

Comment:
This `popd` won't even be executed since you `exit` on both `if` branches above. You can simply remove it.